platform/lib/routers/exampleEmbeds.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
suggestion: move the sample routers into a `example` subdir to avoid duplicating the example prefix in the file name.